src/components/Dashboard/Dashboard.js

```diff
@@ -1,17 +1,13 @@
-import React, { useContext } from 'react'
-import Card from '../Card/Card'
-import './Dashboard.scss'
-import UsersContext from '../../Context/UsersContext'
+import React, { useContext } from "react"
+import Card from "../Card/Card"
```

Comment:
I see that we replaced single quotations with double quotations, we will have to go through the application at the end and make sure we are consistent with our syntax